app/javascript/githubRepos/githubRepos.jsx

```diff
@@ -0,0 +1,60 @@
+import { h, Component } from 'preact';
+import { SingleRepo } from './singleRepo';
+
+export class GithubRepos extends Component {
+  constructor(props) {
```

Comment:
@Zhao-Andy, this works, but the constructor is not necessary since you're not actually doing anything with the `props`. You can simply initialize the state like this:

```
export class GithubRepos extends Component {
    state = {
      repos: [],
      erroredOut: false,
    };

    ...
}
```

Reply:
Awesome, just the sort of feedback I was looking for. Will make changes 👍